app/src/pages/playground/ModelConfigButton.tsx

```diff
@@ -14,19 +14,95 @@ import {
   DialogContainer,
   Flex,
   Form,
+  Item,
+  Picker,
   Text,
   TextField,
   View,
 } from "@arizeai/components";
 
-import { ModelProviders } from "@phoenix/constants/generativeConstants";
+import {
+  AzureOpenAiApiVersions,
+  ModelProviders,
+} from "@phoenix/constants/generativeConstants";
 import { usePlaygroundContext } from "@phoenix/contexts/PlaygroundContext";
+import { PlaygroundInstance } from "@phoenix/store";
 
 import { ModelConfigButtonDialogQuery } from "./__generated__/ModelConfigButtonDialogQuery.graphql";
 import { ModelPicker } from "./ModelPicker";
 import { ModelProviderPicker } from "./ModelProviderPicker";
 import { PlaygroundInstanceProps } from "./types";
 
+function AzureOpenAiModelConfigFormField({
```

Comment:
We could make this a form a to be used else where, below is already wrapped in a form that's why i pulled out the fields like this, can refactor if we wan tthis to be reusable